src/ol/renderer/Composite.js

```diff
@@ -30,9 +30,13 @@ class CompositeMapRenderer extends MapRenderer {
      * @type {HTMLDivElement}
      */
     this.element_ = document.createElement('div');
+    const style = this.element_.style;
+    style.display = 'flex';
```

Comment:
we will lose ie < 10 support (do we still care ?)

Reply:
As discussed, if someone comes along with funds for IE 9 support, we can implement this in a different way.